boards/qemu_cortex_a53.overlay

```diff
@@ -49,6 +49,14 @@
 			};
 		};
 	};
+
+	firmware {
+		tee {
+		compatible = "linaro,optee-tz";
```

Comment:
looks like it should be align with block, i.e. add tab here and for all lines int the same section/scope
the same in other overlay files

Reply:
qemu board well be deleted